circuit/types/group/src/helpers/from_unchecked.rs

```diff
@@ -0,0 +1,75 @@
+// Copyright (C) 2019-2022 Aleo Systems Inc.
+// This file is part of the snarkVM library.
+
+// The snarkVM library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// The snarkVM library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with the snarkVM library. If not, see <https://www.gnu.org/licenses/>.
+
+use super::*;
+
+impl<E: Environment> Group<E> {
+    /// Initializes an affine group element from a given x- and y-coordinate field element.
+    /// Note: The resulting point is **not** enforced to be on the curve with constraints.
+    #[doc(hidden)]
+    #[cfg(any(test, feature = "unchecked"))]
```

Comment:
Why are we feature-gating this? Isn't it always on (in our code base), since `algorithms` requires it?

Reply:
I've removed the "unchecked" feature flag.